recipe/meta.yaml

```diff
@@ -6,19 +6,20 @@ package:
 
 source:
   url: https://github.com/SciTools/nc-time-axis/archive/v{{ version }}.tar.gz
-  sha256: 49c4b1022e601e22a2fb5fc6fc2af69070cceb5cdd01a880ae7379caef8f2f90
+  sha256: 2fae9b904a5284581aa8ac1fd954776fc246339665411724b23d0e3b4449332f
```

Comment:
I've no idea why the sha for the same binary blob may have changed... but we're about to push `1.3.0`, so it's a soon to be bit of a moot point